install.js

```diff
@@ -150,7 +152,7 @@ function findSuitableTempDirectory() {
 
 
 function getRequestOptions(downloadPath) {
-  const options = {uri: downloadPath, method: 'GET'};
+  const options = { uri: downloadPath, method: 'GET' };
```

Comment:
Please do not reformat the file, remove all white space diffs.

Reply:
Wow. I didn't even notice that. Sorry.

Reply:
I would consider turning off https://github.com/giggio/node-chromedriver/blob/master/.vscode/settings.json#L3. It took me forever to figure out why the file was auto formatting.

Reply:
I've been using it on Vim, the problem is probably on the ES Lint config. https://github.com/giggio/node-chromedriver/blob/master/.eslintrc.json
This was supposed to be helping... I'll check it out. Thanks for the info.

Reply:
@kellyselden I have just tried linting the files and there was no reformating. The .eslintrc.json a .editorconfig files seem fine. Do you think it could be some other extension you have?